src/pyOpenMS/pxds/SplineInterpolatedPeaks.pxd

```diff
@@ -20,17 +20,17 @@ cdef extern from "<OpenMS/FILTERING/DATAREDUCTION/SplineInterpolatedPeaks.h>" na
 
         int size() nogil except +
         
-        SplineInterpolatedPeaks_Navigator getNavigator(double scaling) nogil except +
+        Navigator getNavigator(double scaling) nogil except +
 
 
 cdef extern from "<OpenMS/FILTERING/DATAREDUCTION/SplineInterpolatedPeaks.h>" namespace "OpenMS::SplineInterpolatedPeaks":
     
-    cdef cppclass SplineSpectrum_Navigator "OpenMS::SplineInterpolatedPeaks::Navigator":
+    cdef cppclass Navigator:
```

Comment:
This does mean that "Navigator" is now the in the global namespace of pyOpenMS. I think `SplineSpectrum_Navigator` would be better